test failures

Gentoo CI had this test failures
```
=================================== FAILURES ===================================
_________ test_designspace_generation_regular_same_family_name[defcon] _________

tmpdir = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re0')
ufo_module = <module 'defcon' from '/usr/lib/python3.8/site-packages/defcon/__init__.py'>

    def test_designspace_generation_regular_same_family_name(tmpdir, ufo_module):
        ufo_Lt = ufo_module.Font()
        ufo_Lt.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Lt.info.styleName = "Light"
        ufo_Lt.info.openTypeOS2WeightClass = 300
    
        ufo_Rg = ufo_module.Font()
        ufo_Rg.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Rg.info.styleName = "Regular"
        ufo_Rg.info.openTypeOS2WeightClass = 400
    
        ufo_Md = ufo_module.Font()
        ufo_Md.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Md.info.styleName = "Medium"
        ufo_Md.info.openTypeOS2WeightClass = 500
    
        ufo_Bd = ufo_module.Font()
        ufo_Bd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Bd.info.styleName = "Bold"
        ufo_Bd.info.openTypeOS2WeightClass = 700
    
        ufo_ExBd = ufo_module.Font()
        ufo_ExBd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_ExBd.info.styleName = "XBold"
        ufo_ExBd.info.openTypeOS2WeightClass = 800
    
        font = to_glyphs([ufo_Lt, ufo_Rg, ufo_Md, ufo_Bd, ufo_ExBd])
        designspace = to_designspace(font, ufo_module=ufo_module)
    
        path = os.path.join(str(tmpdir), "actual.designspace")
        designspace.write(path)
    
        expected_path = os.path.join(
            os.path.dirname(__file__), "..", "data", "DesignspaceGenTestRegular.designspace"
        )
    
>       assert (
            len(main.diff_files(path, expected_path, formatter=formatting.DiffFormatter()))
            == 0
        )
E       assert 50 == 0
E        +  where 50 = len('[update-attribute, /designspace[1], format, "4.1"]')
E        +    where '[update-attribute, /designspace[1], format, "4.1"]' = <function diff_files at 0x7f6c2b3b8820>('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re0/actual.designspace', '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestRegular.designspace', formatter=<xmldiff.formatting.DiffFormatter object at 0x7f6c2adad040>)
E        +      where <function diff_files at 0x7f6c2b3b8820> = main.diff_files
E        +      and   <xmldiff.formatting.DiffFormatter object at 0x7f6c2adad040> = <class 'xmldiff.formatting.DiffFormatter'>()
E        +        where <class 'xmldiff.formatting.DiffFormatter'> = formatting.DiffFormatter

designspace = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2adad610>
expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestRegular.designspace'
font       = <GSFont "CoolFoundry Examplary Serif">
path       = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re0/actual.designspace'
tmpdir     = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re0')
ufo_Bd     = <defcon.objects.font.Font object at 0x7f6c2b5252b0>
ufo_ExBd   = <defcon.objects.font.Font object at 0x7f6c2b525dc0>
ufo_Lt     = <defcon.objects.font.Font object at 0x7f6c2adad460>
ufo_Md     = <defcon.objects.font.Font object at 0x7f6c2b525fd0>
ufo_Rg     = <defcon.objects.font.Font object at 0x7f6c2ad764c0>
ufo_module = <module 'defcon' from '/usr/lib/python3.8/site-packages/defcon/__init__.py'>

tests/builder/designspace_gen_test.py:69: AssertionError
_________ test_designspace_generation_italic_same_family_name[defcon] __________

tmpdir = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it0')
ufo_module = <module 'defcon' from '/usr/lib/python3.8/site-packages/defcon/__init__.py'>

    def test_designspace_generation_italic_same_family_name(tmpdir, ufo_module):
        ufo_Lt = ufo_module.Font()
        ufo_Lt.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Lt.info.styleName = "Light Italic"
        ufo_Lt.info.openTypeOS2WeightClass = 300
        ufo_Lt.info.italicAngle = -11
    
        ufo_Rg = ufo_module.Font()
        ufo_Rg.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Rg.info.styleName = "Regular Italic"
        ufo_Rg.info.openTypeOS2WeightClass = 400
        ufo_Rg.info.italicAngle = -11
    
        ufo_Md = ufo_module.Font()
        ufo_Md.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Md.info.styleName = "Medium Italic"
        ufo_Md.info.openTypeOS2WeightClass = 500
        ufo_Md.info.italicAngle = -11
    
        ufo_Bd = ufo_module.Font()
        ufo_Bd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Bd.info.styleName = "Bold Italic"
        ufo_Bd.info.openTypeOS2WeightClass = 700
        ufo_Bd.info.italicAngle = -11
    
        ufo_ExBd = ufo_module.Font()
        ufo_ExBd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_ExBd.info.styleName = "XBold Italic"
        ufo_ExBd.info.openTypeOS2WeightClass = 800
        ufo_ExBd.info.italicAngle = -11
    
        font = to_glyphs([ufo_Lt, ufo_Rg, ufo_Md, ufo_Bd, ufo_ExBd])
        designspace = to_designspace(font, ufo_module=ufo_module)
    
        path = os.path.join(str(tmpdir), "actual.designspace")
        designspace.write(path)
    
        expected_path = os.path.join(
            os.path.dirname(__file__), "..", "data", "DesignspaceGenTestItalic.designspace"
        )
    
>       assert (
            len(main.diff_files(path, expected_path, formatter=formatting.DiffFormatter()))
            == 0
        )
E       assert 50 == 0
E        +  where 50 = len('[update-attribute, /designspace[1], format, "4.1"]')
E        +    where '[update-attribute, /designspace[1], format, "4.1"]' = <function diff_files at 0x7f6c2b3b8820>('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it0/actual.designspace', '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestItalic.designspace', formatter=<xmldiff.formatting.DiffFormatter object at 0x7f6c2b2a79d0>)
E        +      where <function diff_files at 0x7f6c2b3b8820> = main.diff_files
E        +      and   <xmldiff.formatting.DiffFormatter object at 0x7f6c2b2a79d0> = <class 'xmldiff.formatting.DiffFormatter'>()
E        +        where <class 'xmldiff.formatting.DiffFormatter'> = formatting.DiffFormatter

designspace = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2b2a75e0>
expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestItalic.designspace'
font       = <GSFont "CoolFoundry Examplary Serif">
path       = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it0/actual.designspace'
tmpdir     = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it0')
ufo_Bd     = <defcon.objects.font.Font object at 0x7f6c2b1d5e20>
ufo_ExBd   = <defcon.objects.font.Font object at 0x7f6c2b1d5970>
ufo_Lt     = <defcon.objects.font.Font object at 0x7f6c2b1d5550>
ufo_Md     = <defcon.objects.font.Font object at 0x7f6c2b1d5400>
ufo_Rg     = <defcon.objects.font.Font object at 0x7f6c2b1d5310>
ufo_module = <module 'defcon' from '/usr/lib/python3.8/site-packages/defcon/__init__.py'>

tests/builder/designspace_gen_test.py:116: AssertionError
________ test_designspace_generation_regular_same_family_name[ufoLib2] _________

tmpdir = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re2')
ufo_module = <module 'ufoLib2' from '/usr/lib/python3.8/site-packages/ufoLib2/__init__.py'>

    def test_designspace_generation_regular_same_family_name(tmpdir, ufo_module):
        ufo_Lt = ufo_module.Font()
        ufo_Lt.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Lt.info.styleName = "Light"
        ufo_Lt.info.openTypeOS2WeightClass = 300
    
        ufo_Rg = ufo_module.Font()
        ufo_Rg.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Rg.info.styleName = "Regular"
        ufo_Rg.info.openTypeOS2WeightClass = 400
    
        ufo_Md = ufo_module.Font()
        ufo_Md.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Md.info.styleName = "Medium"
        ufo_Md.info.openTypeOS2WeightClass = 500
    
        ufo_Bd = ufo_module.Font()
        ufo_Bd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Bd.info.styleName = "Bold"
        ufo_Bd.info.openTypeOS2WeightClass = 700
    
        ufo_ExBd = ufo_module.Font()
        ufo_ExBd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_ExBd.info.styleName = "XBold"
        ufo_ExBd.info.openTypeOS2WeightClass = 800
    
        font = to_glyphs([ufo_Lt, ufo_Rg, ufo_Md, ufo_Bd, ufo_ExBd])
        designspace = to_designspace(font, ufo_module=ufo_module)
    
        path = os.path.join(str(tmpdir), "actual.designspace")
        designspace.write(path)
    
        expected_path = os.path.join(
            os.path.dirname(__file__), "..", "data", "DesignspaceGenTestRegular.designspace"
        )
    
>       assert (
            len(main.diff_files(path, expected_path, formatter=formatting.DiffFormatter()))
            == 0
        )
E       assert 50 == 0
E        +  where 50 = len('[update-attribute, /designspace[1], format, "4.1"]')
E        +    where '[update-attribute, /designspace[1], format, "4.1"]' = <function diff_files at 0x7f6c2b3b8820>('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re2/actual.designspace', '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestRegular.designspace', formatter=<xmldiff.formatting.DiffFormatter object at 0x7f6c2ab7f3a0>)
E        +      where <function diff_files at 0x7f6c2b3b8820> = main.diff_files
E        +      and   <xmldiff.formatting.DiffFormatter object at 0x7f6c2ab7f3a0> = <class 'xmldiff.formatting.DiffFormatter'>()
E        +        where <class 'xmldiff.formatting.DiffFormatter'> = formatting.DiffFormatter

designspace = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2ab7fc70>
expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestRegular.designspace'
font       = <GSFont "CoolFoundry Examplary Serif">
path       = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re2/actual.designspace'
tmpdir     = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_re2')
ufo_Bd     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Bold' at 0x7f6c27a00820>
ufo_ExBd   = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif XBold' at 0x7f6c27a008b0>
ufo_Lt     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Light' at 0x7f6c27a003a0>
ufo_Md     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Medium' at 0x7f6c27a00790>
ufo_Rg     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Regular' at 0x7f6c27a00700>
ufo_module = <module 'ufoLib2' from '/usr/lib/python3.8/site-packages/ufoLib2/__init__.py'>

tests/builder/designspace_gen_test.py:69: AssertionError
_________ test_designspace_generation_italic_same_family_name[ufoLib2] _________

tmpdir = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it1')
ufo_module = <module 'ufoLib2' from '/usr/lib/python3.8/site-packages/ufoLib2/__init__.py'>

    def test_designspace_generation_italic_same_family_name(tmpdir, ufo_module):
        ufo_Lt = ufo_module.Font()
        ufo_Lt.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Lt.info.styleName = "Light Italic"
        ufo_Lt.info.openTypeOS2WeightClass = 300
        ufo_Lt.info.italicAngle = -11
    
        ufo_Rg = ufo_module.Font()
        ufo_Rg.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Rg.info.styleName = "Regular Italic"
        ufo_Rg.info.openTypeOS2WeightClass = 400
        ufo_Rg.info.italicAngle = -11
    
        ufo_Md = ufo_module.Font()
        ufo_Md.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Md.info.styleName = "Medium Italic"
        ufo_Md.info.openTypeOS2WeightClass = 500
        ufo_Md.info.italicAngle = -11
    
        ufo_Bd = ufo_module.Font()
        ufo_Bd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_Bd.info.styleName = "Bold Italic"
        ufo_Bd.info.openTypeOS2WeightClass = 700
        ufo_Bd.info.italicAngle = -11
    
        ufo_ExBd = ufo_module.Font()
        ufo_ExBd.info.familyName = "CoolFoundry Examplary Serif"
        ufo_ExBd.info.styleName = "XBold Italic"
        ufo_ExBd.info.openTypeOS2WeightClass = 800
        ufo_ExBd.info.italicAngle = -11
    
        font = to_glyphs([ufo_Lt, ufo_Rg, ufo_Md, ufo_Bd, ufo_ExBd])
        designspace = to_designspace(font, ufo_module=ufo_module)
    
        path = os.path.join(str(tmpdir), "actual.designspace")
        designspace.write(path)
    
        expected_path = os.path.join(
            os.path.dirname(__file__), "..", "data", "DesignspaceGenTestItalic.designspace"
        )
    
>       assert (
            len(main.diff_files(path, expected_path, formatter=formatting.DiffFormatter()))
            == 0
        )
E       assert 50 == 0
E        +  where 50 = len('[update-attribute, /designspace[1], format, "4.1"]')
E        +    where '[update-attribute, /designspace[1], format, "4.1"]' = <function diff_files at 0x7f6c2b3b8820>('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it1/actual.designspace', '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestItalic.designspace', formatter=<xmldiff.formatting.DiffFormatter object at 0x7f6c2b02aeb0>)
E        +      where <function diff_files at 0x7f6c2b3b8820> = main.diff_files
E        +      and   <xmldiff.formatting.DiffFormatter object at 0x7f6c2b02aeb0> = <class 'xmldiff.formatting.DiffFormatter'>()
E        +        where <class 'xmldiff.formatting.DiffFormatter'> = formatting.DiffFormatter

designspace = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2adbf8b0>
expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceGenTestItalic.designspace'
font       = <GSFont "CoolFoundry Examplary Serif">
path       = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it1/actual.designspace'
tmpdir     = local('/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/pytest-of-portage/pytest-0/test_designspace_generation_it1')
ufo_Bd     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Bold Italic' at 0x7f6c27b5d5e0>
ufo_ExBd   = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif XBold Italic' at 0x7f6c27b5d670>
ufo_Lt     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Light Italic' at 0x7f6c27a00040>
ufo_Md     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Medium Italic' at 0x7f6c27b5d550>
ufo_Rg     = <ufoLib2.objects.font.Font 'CoolFoundry Examplary Serif Regular Italic' at 0x7f6c27b5d4c0>
ufo_module = <module 'ufoLib2' from '/usr/lib/python3.8/site-packages/ufoLib2/__init__.py'>

tests/builder/designspace_gen_test.py:116: AssertionError
__________________________ DesignspaceTest.test_basic __________________________

self = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_basic>

    def test_basic(self):
        masters, instances = makeFamily()
        font = makeFont(masters, instances, "DesignspaceTest Basic")
        doc = to_designspace(font, instance_dir="out")
>       self.expect_designspace(doc, "DesignspaceTestBasic.designspace")

doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2adce640>
font       = <GSFont "DesignspaceTest Basic">
instances  = [<GSInstance >, <GSInstance >, <GSInstance >, <GSInstance >]
masters    = [<GSFontMaster "Regular" width 100 weight 90.0>,
 <GSFontMaster "Black" width 100 weight 190.0>]
self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_basic>

tests/builder/interpolation_test.py:179: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/builder/interpolation_test.py:155: in expect_designspace
    return self._expect_designspace(doc, expected_path)
        dirname    = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2adce640>
        expected_name = 'DesignspaceTestBasic.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestBasic.designspace'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_basic>
tests/builder/interpolation_test.py:167: in _expect_designspace
    self.fail("*.designspace file is different from expected")
E   AssertionError: *.designspace file is different from expected
        actual_diff = ('[update-attribute, /designspace[1], format, "4.1"]\n'
 '[insert, /designspace/instances/instance[1], kerning, 1]\n'
 '[insert, /designspace/instances/instance[1], info, 2]\n'
 '[insert, /designspace/instances/instance[2], kerning, 1]\n'
 '[insert, /designspace/instances/instance[2], info, 2]\n'
 '[insert, /designspace/instances/instance[3], kerning, 1]\n'
 '[insert, /designspace/instances/instance[3], info, 2]\n'
 '[insert, /designspace/instances/instance[4], kerning, 1]\n'
 '[insert, /designspace/instances/instance[4], info, 2]')
        actual_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/tmp_t5c8kjj/generated.designspace'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2adce640>
        expected_name = 'DesignspaceTestBasic.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestBasic.designspace'
        line       = '[insert, /designspace/instances/instance[4], info, 2]'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_basic>
----------------------------- Captured stderr call -----------------------------
DesignspaceTestBasic.designspace discrepancies (per xmldiff):
  [update-attribute, /designspace[1], format, "4.1"]  [insert, /designspace/instances/instance[1], kerning, 1]  [insert, /designspace/instances/instance[1], info, 2]  [insert, /designspace/instances/instance[2], kerning, 1]  [insert, /designspace/instances/instance[2], info, 2]  [insert, /designspace/instances/instance[3], kerning, 1]  [insert, /designspace/instances/instance[3], info, 2]  [insert, /designspace/instances/instance[4], kerning, 1]  [insert, /designspace/instances/instance[4], info, 2]
_______________________ DesignspaceTest.test_familyName ________________________

self = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_familyName>

    def test_familyName(self):
        masters, _ = makeFamily()
        customFamily = makeInstance("Regular", weight=("Bold", 600, 151))
        customFamily.customParameters["familyName"] = "Custom Family"
        instances = [makeInstance("Regular", weight=("Regular", 400, 90)), customFamily]
        font = makeFont(masters, instances, "DesignspaceTest FamilyName")
        doc = to_designspace(font, instance_dir="out")
>       self.expect_designspace(doc, "DesignspaceTestFamilyName.designspace")

_          = [<GSInstance >, <GSInstance >, <GSInstance >, <GSInstance >]
customFamily = <GSInstance >
doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2a800af0>
font       = <GSFont "DesignspaceTest FamilyName">
instances  = [<GSInstance >, <GSInstance >]
masters    = [<GSFontMaster "Regular" width 100 weight 90.0>,
 <GSFontMaster "Black" width 100 weight 190.0>]
self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_familyName>

tests/builder/interpolation_test.py:206: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/builder/interpolation_test.py:155: in expect_designspace
    return self._expect_designspace(doc, expected_path)
        dirname    = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2a800af0>
        expected_name = 'DesignspaceTestFamilyName.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestFamilyName.designspace'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_familyName>
tests/builder/interpolation_test.py:167: in _expect_designspace
    self.fail("*.designspace file is different from expected")
E   AssertionError: *.designspace file is different from expected
        actual_diff = ('[update-attribute, /designspace[1], format, "4.1"]\n'
 '[insert, /designspace/instances/instance[1], kerning, 1]\n'
 '[insert, /designspace/instances/instance[1], info, 2]\n'
 '[insert, /designspace/instances/instance[2], kerning, 1]\n'
 '[insert, /designspace/instances/instance[2], info, 2]')
        actual_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/tmpcgqgo5z7/generated.designspace'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2a800af0>
        expected_name = 'DesignspaceTestFamilyName.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestFamilyName.designspace'
        line       = '[insert, /designspace/instances/instance[2], info, 2]'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_familyName>
----------------------------- Captured stderr call -----------------------------
DesignspaceTestFamilyName.designspace discrepancies (per xmldiff):
  [update-attribute, /designspace[1], format, "4.1"]  [insert, /designspace/instances/instance[1], kerning, 1]  [insert, /designspace/instances/instance[1], info, 2]  [insert, /designspace/instances/instance[2], kerning, 1]  [insert, /designspace/instances/instance[2], info, 2]
________________________ DesignspaceTest.test_fileName _________________________

self = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_fileName>

    def test_fileName(self):
        masters, _ = makeFamily()
        customFileName = makeInstance("Regular", weight=("Bold", 600, 151))
        customFileName.customParameters["fileName"] = "Custom FileName"
        instances = [
            makeInstance("Regular", weight=("Regular", 400, 90)),
            customFileName,
        ]
        font = makeFont(masters, instances, "DesignspaceTest FamilyName")
        doc = to_designspace(font, instance_dir="out")
>       self.expect_designspace(doc, "DesignspaceTestFileName.designspace")

_          = [<GSInstance >, <GSInstance >, <GSInstance >, <GSInstance >]
customFileName = <GSInstance >
doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2b5023a0>
font       = <GSFont "DesignspaceTest FamilyName">
instances  = [<GSInstance >, <GSInstance >]
masters    = [<GSFontMaster "Regular" width 100 weight 90.0>,
 <GSFontMaster "Black" width 100 weight 190.0>]
self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_fileName>

tests/builder/interpolation_test.py:219: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/builder/interpolation_test.py:155: in expect_designspace
    return self._expect_designspace(doc, expected_path)
        dirname    = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2b5023a0>
        expected_name = 'DesignspaceTestFileName.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestFileName.designspace'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_fileName>
tests/builder/interpolation_test.py:167: in _expect_designspace
    self.fail("*.designspace file is different from expected")
E   AssertionError: *.designspace file is different from expected
        actual_diff = ('[update-attribute, /designspace[1], format, "4.1"]\n'
 '[insert, /designspace/instances/instance[1], kerning, 1]\n'
 '[insert, /designspace/instances/instance[1], info, 2]\n'
 '[insert, /designspace/instances/instance[2], kerning, 1]\n'
 '[insert, /designspace/instances/instance[2], info, 2]')
        actual_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/tmpnwrtm_vq/generated.designspace'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2b5023a0>
        expected_name = 'DesignspaceTestFileName.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestFileName.designspace'
        line       = '[insert, /designspace/instances/instance[2], info, 2]'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_fileName>
----------------------------- Captured stderr call -----------------------------
DesignspaceTestFileName.designspace discrepancies (per xmldiff):
  [update-attribute, /designspace[1], format, "4.1"]  [insert, /designspace/instances/instance[1], kerning, 1]  [insert, /designspace/instances/instance[1], info, 2]  [insert, /designspace/instances/instance[2], kerning, 1]  [insert, /designspace/instances/instance[2], info, 2]
__________________ DesignspaceTest.test_inactive_from_exports __________________

self = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_inactive_from_exports>

    def test_inactive_from_exports(self):
        # Glyphs.app recognizes exports=0 as a flag for inactive instances.
        # https://github.com/googlefonts/glyphsLib/issues/129
        masters, instances = makeFamily()
        for inst in instances:
            if inst.name != "Semibold":
                inst.exports = False
        font = makeFont(masters, instances, "DesignspaceTest Inactive")
        doc = to_designspace(font, instance_dir="out")
>       self.expect_designspace(doc, "DesignspaceTestInactive.designspace")

doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c28cf71f0>
font       = <GSFont "DesignspaceTest Inactive">
inst       = <GSInstance >
instances  = [<GSInstance >, <GSInstance >, <GSInstance >, <GSInstance >]
masters    = [<GSFontMaster "Regular" width 100 weight 90.0>,
 <GSFontMaster "Black" width 100 weight 190.0>]
self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_inactive_from_exports>

tests/builder/interpolation_test.py:191: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/builder/interpolation_test.py:155: in expect_designspace
    return self._expect_designspace(doc, expected_path)
        dirname    = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c28cf71f0>
        expected_name = 'DesignspaceTestInactive.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestInactive.designspace'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_inactive_from_exports>
tests/builder/interpolation_test.py:167: in _expect_designspace
    self.fail("*.designspace file is different from expected")
E   AssertionError: *.designspace file is different from expected
        actual_diff = ('[update-attribute, /designspace[1], format, "4.1"]\n'
 '[insert, /designspace/instances/instance[1], kerning, 1]\n'
 '[insert, /designspace/instances/instance[1], info, 2]')
        actual_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/tmp818qdxfp/generated.designspace'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c28cf71f0>
        expected_name = 'DesignspaceTestInactive.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestInactive.designspace'
        line       = '[insert, /designspace/instances/instance[1], info, 2]'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_inactive_from_exports>
----------------------------- Captured stderr call -----------------------------
DesignspaceTestInactive.designspace discrepancies (per xmldiff):
  [update-attribute, /designspace[1], format, "4.1"]  [insert, /designspace/instances/instance[1], kerning, 1]  [insert, /designspace/instances/instance[1], info, 2]
______________________ DesignspaceTest.test_instanceOrder ______________________

self = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_instanceOrder>

    def test_instanceOrder(self):
        # The generated *.designspace file should place instances
        # in the same order as they appear in the original source.
        # https://github.com/googlefonts/glyphsLib/issues/113
        masters, _ = makeFamily()
        instances = [
            makeInstance("Black", weight=("Black", 900, 190)),
            makeInstance("Regular", weight=("Regular", 400, 90)),
            makeInstance("Bold", weight=("Bold", 700, 151), is_bold=True),
        ]
        font = makeFont(masters, instances, "DesignspaceTest InstanceOrder")
        doc = to_designspace(font, instance_dir="out")
>       self.expect_designspace(doc, "DesignspaceTestInstanceOrder.designspace")

_          = [<GSInstance >, <GSInstance >, <GSInstance >, <GSInstance >]
doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2ad9eaf0>
font       = <GSFont "DesignspaceTest InstanceOrder">
instances  = [<GSInstance >, <GSInstance >, <GSInstance >]
masters    = [<GSFontMaster "Regular" width 100 weight 90.0>,
 <GSFontMaster "Black" width 100 weight 190.0>]
self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_instanceOrder>

tests/builder/interpolation_test.py:274: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/builder/interpolation_test.py:155: in expect_designspace
    return self._expect_designspace(doc, expected_path)
        dirname    = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2ad9eaf0>
        expected_name = 'DesignspaceTestInstanceOrder.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestInstanceOrder.designspace'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_instanceOrder>
tests/builder/interpolation_test.py:167: in _expect_designspace
    self.fail("*.designspace file is different from expected")
E   AssertionError: *.designspace file is different from expected
        actual_diff = ('[update-attribute, /designspace[1], format, "4.1"]\n'
 '[insert, /designspace/instances/instance[1], kerning, 1]\n'
 '[insert, /designspace/instances/instance[1], info, 2]\n'
 '[insert, /designspace/instances/instance[2], kerning, 1]\n'
 '[insert, /designspace/instances/instance[2], info, 2]\n'
 '[insert, /designspace/instances/instance[3], kerning, 1]\n'
 '[insert, /designspace/instances/instance[3], info, 2]')
        actual_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/tmp1f1w8x6y/generated.designspace'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2ad9eaf0>
        expected_name = 'DesignspaceTestInstanceOrder.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestInstanceOrder.designspace'
        line       = '[insert, /designspace/instances/instance[3], info, 2]'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_instanceOrder>
----------------------------- Captured stderr call -----------------------------
DesignspaceTestInstanceOrder.designspace discrepancies (per xmldiff):
  [update-attribute, /designspace[1], format, "4.1"]  [insert, /designspace/instances/instance[1], kerning, 1]  [insert, /designspace/instances/instance[1], info, 2]  [insert, /designspace/instances/instance[2], kerning, 1]  [insert, /designspace/instances/instance[2], info, 2]  [insert, /designspace/instances/instance[3], kerning, 1]  [insert, /designspace/instances/instance[3], info, 2]
_________________________ DesignspaceTest.test_twoAxes _________________________

self = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_twoAxes>

    def test_twoAxes(self):
        # In NotoSansArabic-MM.glyphs, the regular width only contains
        # parameters for the weight axis. For the width axis, glyphsLib
        # should use 100 as default value (just like Glyphs.app does).
        familyName = "DesignspaceTest TwoAxes"
        masters = [
            makeMaster("Regular", weight=90),
            makeMaster("Black", weight=190),
            makeMaster("Thin", weight=26),
            makeMaster("ExtraCond", weight=90, width=70),
            makeMaster("ExtraCond Black", weight=190, width=70),
            makeMaster("ExtraCond Thin", weight=26, width=70),
        ]
        instances = [
            makeInstance("Thin", weight=("Thin", 100, 26)),
            makeInstance("Regular", weight=("Regular", 400, 90)),
            makeInstance("Semibold", weight=("SemiBold", 600, 128)),
            makeInstance("Black", weight=("Black", 900, 190)),
            makeInstance(
                "ExtraCondensed Thin",
                weight=("Thin", 100, 26),
                width=("Extra Condensed", 2, 70),
            ),
            makeInstance(
                "ExtraCondensed",
                weight=("Regular", 400, 90),
                width=("Extra Condensed", 2, 70),
            ),
            makeInstance(
                "ExtraCondensed Black",
                weight=("Black", 900, 190),
                width=("Extra Condensed", 2, 70),
            ),
        ]
        font = makeFont(masters, instances, familyName)
        doc = to_designspace(font, instance_dir="out")
>       self.expect_designspace(doc, "DesignspaceTestTwoAxes.designspace")

doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2abd5970>
familyName = 'DesignspaceTest TwoAxes'
font       = <GSFont "DesignspaceTest TwoAxes">
instances  = [<GSInstance >,
 <GSInstance >,
 <GSInstance >,
 <GSInstance >,
 <GSInstance >,
 <GSInstance >,
 <GSInstance >]
masters    = [<GSFontMaster "Regular" width 100 weight 90>,
 <GSFontMaster "Black" width 100 weight 190>,
 <GSFontMaster "Thin" width 100 weight 26>,
 <GSFontMaster "ExtraCond" width 70 weight 90>,
 <GSFontMaster "ExtraCond Black" width 70 weight 190>,
 <GSFontMaster "ExtraCond Thin" width 70 weight 26>]
self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_twoAxes>

tests/builder/interpolation_test.py:313: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/builder/interpolation_test.py:155: in expect_designspace
    return self._expect_designspace(doc, expected_path)
        dirname    = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2abd5970>
        expected_name = 'DesignspaceTestTwoAxes.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestTwoAxes.designspace'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_twoAxes>
tests/builder/interpolation_test.py:167: in _expect_designspace
    self.fail("*.designspace file is different from expected")
E   AssertionError: *.designspace file is different from expected
        actual_diff = ('[update-attribute, /designspace[1], format, "4.1"]\n'
 '[update-attribute, /designspace/axes/axis[2], minimum, "62.500000"]\n'
 '[update-attribute, /designspace/axes/axis[2]/map[1], input, "62.500000"]\n'
 '[insert, /designspace/instances/instance[1], kerning, 1]\n'
 '[insert, /designspace/instances/instance[1], info, 2]\n'
 '[insert, /designspace/instances/instance[2], kerning, 1]\n'
 '[insert, /designspace/instances/instance[2], info, 2]\n'
 '[insert, /designspace/instances/instance[3], kerning, 1]\n'
 '[insert, /designspace/instances/instance[3], info, 2]\n'
 '[insert, /designspace/instances/instance[4], kerning, 1]\n'
 '[insert, /designspace/instances/instance[4], info, 2]\n'
 '[insert, /designspace/instances/instance[5], kerning, 1]\n'
 '[insert, /designspace/instances/instance[5], info, 2]\n'
 '[insert, /designspace/instances/instance[6], kerning, 1]\n'
 '[insert, /designspace/instances/instance[6], info, 2]\n'
 '[insert, /designspace/instances/instance[7], kerning, 1]\n'
 '[insert, /designspace/instances/instance[7], info, 2]')
        actual_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/temp/tmp0r02nns9/generated.designspace'
        doc        = <fontTools.designspaceLib.DesignSpaceDocument object at 0x7f6c2abd5970>
        expected_name = 'DesignspaceTestTwoAxes.designspace'
        expected_path = '/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/builder/../data/DesignspaceTestTwoAxes.designspace'
        line       = '[insert, /designspace/instances/instance[7], info, 2]'
        self       = <tests.builder.interpolation_test.DesignspaceTest testMethod=test_twoAxes>
----------------------------- Captured stderr call -----------------------------
DesignspaceTestTwoAxes.designspace discrepancies (per xmldiff):
  [update-attribute, /designspace[1], format, "4.1"]  [update-attribute, /designspace/axes/axis[2], minimum, "62.500000"]  [update-attribute, /designspace/axes/axis[2]/map[1], input, "62.500000"]  [insert, /designspace/instances/instance[1], kerning, 1]  [insert, /designspace/instances/instance[1], info, 2]  [insert, /designspace/instances/instance[2], kerning, 1]  [insert, /designspace/instances/instance[2], info, 2]  [insert, /designspace/instances/instance[3], kerning, 1]  [insert, /designspace/instances/instance[3], info, 2]  [insert, /designspace/instances/instance[4], kerning, 1]  [insert, /designspace/instances/instance[4], info, 2]  [insert, /designspace/instances/instance[5], kerning, 1]  [insert, /designspace/instances/instance[5], info, 2]  [insert, /designspace/instances/instance[6], kerning, 1]  [insert, /designspace/instances/instance[6], info, 2]  [insert, /designspace/instances/instance[7], kerning, 1]  [insert, /designspace/instances/instance[7], info, 2]
=============================== warnings summary ===============================
tests/main_test.py::test_parser_main
  /var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/main_test.py:65: ResourceWarning: unclosed file <_io.TextIOWrapper name='/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/data/GlyphsUnitTestSans.glyphs' mode='r' encoding='utf-8'>
    glyphsLib.parser.main([filename])
  Enable tracemalloc to get traceback where the object was allocated.
  See https://docs.pytest.org/en/stable/how-to/capture-warnings.html#resource-warnings for more info.

tests/main_test.py::test_parser_main_v3
  /var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/main_test.py:80: ResourceWarning: unclosed file <_io.TextIOWrapper name='/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/data/GlyphsUnitTestSans3.glyphs' mode='r' encoding='utf-8'>
    glyphsLib.parser.main([filename])
  Enable tracemalloc to get traceback where the object was allocated.
  See https://docs.pytest.org/en/stable/how-to/capture-warnings.html#resource-warnings for more info.

tests/main_test.py::test_parser_main_v3_upstream
  /var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/main_test.py:90: ResourceWarning: unclosed file <_io.TextIOWrapper name='/var/tmp/portage/dev-python/glyphsLib-6.0.5/work/glyphsLib-6.0.5/tests/data/GlyphsFileFormatv3.glyphs' mode='r' encoding='utf-8'>
    glyphsLib.parser.main([filename])
  Enable tracemalloc to get traceback where the object was allocated.
  See https://docs.pytest.org/en/stable/how-to/capture-warnings.html#resource-warnings for more info.

tests/builder/builder_test.py::ToUfosTestUfoLib2::test_glyph_color_layers_component
tests/builder/builder_test.py::ToUfosTestUfoLib2::test_glyph_color_layers_decompose
tests/builder/builder_test.py::ToUfosTestUfoLib2::test_glyph_color_layers_stroke_no_attributes
tests/builder/builder_test.py::ToUfosTestUfoLib2::test_glyph_color_layers_strokecolor
tests/builder/builder_test.py::ToUfosTestUfoLib2::test_glyph_color_layers_strokewidth
  /usr/lib/python3.8/site-packages/ufoLib2/objects/contour.py:177: UserWarning: The pointPen needs an identifier kwarg. Identifiers have been discarded.
    warnings.warn(

tests/builder/builder_test.py::ToUfosTestDefcon::test_glyph_color_layers_component
tests/builder/builder_test.py::ToUfosTestDefcon::test_glyph_color_layers_decompose
tests/builder/builder_test.py::ToUfosTestDefcon::test_glyph_color_layers_stroke_no_attributes
tests/builder/builder_test.py::ToUfosTestDefcon::test_glyph_color_layers_strokecolor
tests/builder/builder_test.py::ToUfosTestDefcon::test_glyph_color_layers_strokewidth
  /usr/lib/python3.8/site-packages/defcon/objects/contour.py:658: DeprecationWarning: The beginPath method needs an identifier kwarg. The contour's identifier value has been discarded.
    warn("The beginPath method needs an identifier kwarg. The contour's identifier value has been discarded.", DeprecationWarning)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
SKIPPED [2] tests/builder/fontinfo_test.py:432: unconditional skip
SKIPPED [2] tests/builder/to_glyphs_test.py:31: unconditional skip
SKIPPED [2] tests/builder/to_glyphs_test.py:51: unconditional skip
SKIPPED [2] tests/builder/to_glyphs_test.py:72: unconditional skip
SKIPPED [2] tests/builder/to_glyphs_test.py:93: unconditional skip
SKIPPED [2] tests/builder/to_glyphs_test.py:407: Should be better defined
SKIPPED [50] tests/regression_test.py:58: need --run-regression-tests option to run
SKIPPED [5] tests/regression_test.py:77: need --run-regression-tests option to run
FAILED tests/builder/designspace_gen_test.py::test_designspace_generation_regular_same_family_name[defcon]
FAILED tests/builder/designspace_gen_test.py::test_designspace_generation_italic_same_family_name[defcon]
FAILED tests/builder/designspace_gen_test.py::test_designspace_generation_regular_same_family_name[ufoLib2]
FAILED tests/builder/designspace_gen_test.py::test_designspace_generation_italic_same_family_name[ufoLib2]
FAILED tests/builder/interpolation_test.py::DesignspaceTest::test_basic - Ass...
FAILED tests/builder/interpolation_test.py::DesignspaceTest::test_familyName
FAILED tests/builder/interpolation_test.py::DesignspaceTest::test_fileName - ...
FAILED tests/builder/interpolation_test.py::DesignspaceTest::test_inactive_from_exports
FAILED tests/builder/interpolation_test.py::DesignspaceTest::test_instanceOrder
FAILED tests/builder/interpolation_test.py::DesignspaceTest::test_twoAxes - A...
=========== 10 failed, 706 passed, 67 skipped, 13 warnings in 10.09s ===========
```

See https://bugs.gentoo.org/843485
log https://843485.bugs.gentoo.org/attachment.cgi?id=777815

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -32,7 +32,7 @@ flake8==4.0.1
     #   flake8-bugbear
 flake8-bugbear==22.3.23
     # via -r requirements-dev.in
-fonttools[ufo,unicode]==4.32.0
+fonttools[ufo,unicode]==4.34.4
     # via
     #   booleanoperations
     #   cffsubr

--- a/requirements.txt
+++ b/requirements.txt
@@ -8,7 +8,7 @@ appdirs==1.4.4
     # via fs
 attrs==21.4.0
     # via ufolib2
-fonttools[ufo,unicode]==4.32.0
+fonttools[ufo,unicode]==4.34.4
     # via
     #   glyphsLib (setup.cfg)
     #   ufolib2

--- a/setup.cfg
+++ b/setup.cfg
@@ -32,7 +32,7 @@ setup_requires =
     setuptools_scm
 install_requires =
     ufoLib2 >= 0.6.2
-    fonttools[ufo,unicode] >= 4.27.1
+    fonttools[ufo,unicode] >= 4.33.0
     openstep-plist >= 0.3.0
 
 [options.extras_require]

--- a/tests/data/DesignspaceGenTestItalic.designspace
+++ b/tests/data/DesignspaceGenTestItalic.designspace
@@ -1,37 +1,37 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="400" maximum="800" minimum="300" name="Weight" tag="wght" />
-    </axes>
-    <sources>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-LightItalic.ufo" name="CoolFoundry Examplary Serif Light Italic" stylename="Light Italic">
-            <location>
-                <dimension name="Weight" xvalue="300" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-RegularItalic.ufo" name="CoolFoundry Examplary Serif Regular Italic" stylename="Regular Italic">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="400" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-MediumItalic.ufo" name="CoolFoundry Examplary Serif Medium Italic" stylename="Medium Italic">
-            <location>
-                <dimension name="Weight" xvalue="500" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-BoldItalic.ufo" name="CoolFoundry Examplary Serif Bold Italic" stylename="Bold Italic">
-            <location>
-                <dimension name="Weight" xvalue="700" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-XBoldItalic.ufo" name="CoolFoundry Examplary Serif XBold Italic" stylename="XBold Italic">
-            <location>
-                <dimension name="Weight" xvalue="800" />
-            </location>
-        </source>
-    </sources>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="300" maximum="800" default="400"/>
+  </axes>
+  <sources>
+    <source filename="CoolFoundryExamplarySerif-LightItalic.ufo" name="CoolFoundry Examplary Serif Light Italic" familyname="CoolFoundry Examplary Serif" stylename="Light Italic">
+      <location>
+        <dimension name="Weight" xvalue="300"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-RegularItalic.ufo" name="CoolFoundry Examplary Serif Regular Italic" familyname="CoolFoundry Examplary Serif" stylename="Regular Italic">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="400"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-MediumItalic.ufo" name="CoolFoundry Examplary Serif Medium Italic" familyname="CoolFoundry Examplary Serif" stylename="Medium Italic">
+      <location>
+        <dimension name="Weight" xvalue="500"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-BoldItalic.ufo" name="CoolFoundry Examplary Serif Bold Italic" familyname="CoolFoundry Examplary Serif" stylename="Bold Italic">
+      <location>
+        <dimension name="Weight" xvalue="700"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-XBoldItalic.ufo" name="CoolFoundry Examplary Serif XBold Italic" familyname="CoolFoundry Examplary Serif" stylename="XBold Italic">
+      <location>
+        <dimension name="Weight" xvalue="800"/>
+      </location>
+    </source>
+  </sources>
 </designspace>

--- a/tests/data/DesignspaceGenTestRegular.designspace
+++ b/tests/data/DesignspaceGenTestRegular.designspace
@@ -1,37 +1,37 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="400" maximum="800" minimum="300" name="Weight" tag="wght" />
-    </axes>
-    <sources>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-Light.ufo" name="CoolFoundry Examplary Serif Light" stylename="Light">
-            <location>
-                <dimension name="Weight" xvalue="300" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-Regular.ufo" name="CoolFoundry Examplary Serif Regular" stylename="Regular">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="400" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-Medium.ufo" name="CoolFoundry Examplary Serif Medium" stylename="Medium">
-            <location>
-                <dimension name="Weight" xvalue="500" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-Bold.ufo" name="CoolFoundry Examplary Serif Bold" stylename="Bold">
-            <location>
-                <dimension name="Weight" xvalue="700" />
-            </location>
-        </source>
-        <source familyname="CoolFoundry Examplary Serif" filename="CoolFoundryExamplarySerif-XBold.ufo" name="CoolFoundry Examplary Serif XBold" stylename="XBold">
-            <location>
-                <dimension name="Weight" xvalue="800" />
-            </location>
-        </source>
-    </sources>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="300" maximum="800" default="400"/>
+  </axes>
+  <sources>
+    <source filename="CoolFoundryExamplarySerif-Light.ufo" name="CoolFoundry Examplary Serif Light" familyname="CoolFoundry Examplary Serif" stylename="Light">
+      <location>
+        <dimension name="Weight" xvalue="300"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-Regular.ufo" name="CoolFoundry Examplary Serif Regular" familyname="CoolFoundry Examplary Serif" stylename="Regular">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="400"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-Medium.ufo" name="CoolFoundry Examplary Serif Medium" familyname="CoolFoundry Examplary Serif" stylename="Medium">
+      <location>
+        <dimension name="Weight" xvalue="500"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-Bold.ufo" name="CoolFoundry Examplary Serif Bold" familyname="CoolFoundry Examplary Serif" stylename="Bold">
+      <location>
+        <dimension name="Weight" xvalue="700"/>
+      </location>
+    </source>
+    <source filename="CoolFoundryExamplarySerif-XBold.ufo" name="CoolFoundry Examplary Serif XBold" familyname="CoolFoundry Examplary Serif" stylename="XBold">
+      <location>
+        <dimension name="Weight" xvalue="800"/>
+      </location>
+    </source>
+  </sources>
 </designspace>

--- a/tests/data/DesignspaceTestBasic.designspace
+++ b/tests/data/DesignspaceTestBasic.designspace
@@ -1,57 +1,49 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="400" maximum="900" minimum="400" name="Weight" tag="wght">
-            <map input="400" output="90" />
-            <map input="600" output="128" />
-            <map input="700" output="151" />
-            <map input="900" output="190" />
-        </axis>
-    </axes>
-    <sources>
-        <source familyname="DesignspaceTest Basic" filename="DesignspaceTestBasic-Regular.ufo" name="DesignspaceTest Basic Regular" stylename="Regular">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest Basic" filename="DesignspaceTestBasic-Black.ufo" name="DesignspaceTest Basic Black" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-            </location>
-        </source>
-    </sources>
-    <instances>
-        <instance familyname="DesignspaceTest Basic" filename="out/DesignspaceTestBasic-Regular.ufo" name="DesignspaceTest Basic Regular" stylemapfamilyname="DesignspaceTest Basic" stylemapstylename="regular" stylename="Regular">
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest Basic" filename="out/DesignspaceTestBasic-Semibold.ufo" name="DesignspaceTest Basic Semibold" stylemapfamilyname="DesignspaceTest Basic Semibold" stylemapstylename="regular" stylename="Semibold">
-            <location>
-                <dimension name="Weight" xvalue="128" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest Basic" filename="out/DesignspaceTestBasic-Bold.ufo" name="DesignspaceTest Basic Bold" stylemapfamilyname="DesignspaceTest Basic" stylemapstylename="bold" stylename="Bold">
-            <location>
-                <dimension name="Weight" xvalue="151" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest Basic" filename="out/DesignspaceTestBasic-Black.ufo" name="DesignspaceTest Basic Black" stylemapfamilyname="DesignspaceTest Basic Black" stylemapstylename="regular" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-    </instances>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="400" maximum="900" default="400">
+      <map input="400" output="90"/>
+      <map input="600" output="128"/>
+      <map input="700" output="151"/>
+      <map input="900" output="190"/>
+    </axis>
+  </axes>
+  <sources>
+    <source filename="DesignspaceTestBasic-Regular.ufo" name="DesignspaceTest Basic Regular" familyname="DesignspaceTest Basic" stylename="Regular">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestBasic-Black.ufo" name="DesignspaceTest Basic Black" familyname="DesignspaceTest Basic" stylename="Black">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+      </location>
+    </source>
+  </sources>
+  <instances>
+    <instance name="DesignspaceTest Basic Regular" familyname="DesignspaceTest Basic" stylename="Regular" filename="out/DesignspaceTestBasic-Regular.ufo" stylemapfamilyname="DesignspaceTest Basic" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest Basic Semibold" familyname="DesignspaceTest Basic" stylename="Semibold" filename="out/DesignspaceTestBasic-Semibold.ufo" stylemapfamilyname="DesignspaceTest Basic Semibold" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="128"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest Basic Bold" familyname="DesignspaceTest Basic" stylename="Bold" filename="out/DesignspaceTestBasic-Bold.ufo" stylemapfamilyname="DesignspaceTest Basic" stylemapstylename="bold">
+      <location>
+        <dimension name="Weight" xvalue="151"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest Basic Black" familyname="DesignspaceTest Basic" stylename="Black" filename="out/DesignspaceTestBasic-Black.ufo" stylemapfamilyname="DesignspaceTest Basic Black" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+      </location>
+    </instance>
+  </instances>
 </designspace>

--- a/tests/data/DesignspaceTestFamilyName.designspace
+++ b/tests/data/DesignspaceTestFamilyName.designspace
@@ -1,41 +1,37 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="400" maximum="600" minimum="400" name="Weight" tag="wght">
-            <map input="400" output="90" />
-            <map input="600" output="151" />
-        </axis>
-    </axes>
-    <sources>
-        <source familyname="DesignspaceTest FamilyName" filename="DesignspaceTestFamilyName-Regular.ufo" name="DesignspaceTest FamilyName Regular" stylename="Regular">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest FamilyName" filename="DesignspaceTestFamilyName-Black.ufo" name="DesignspaceTest FamilyName Black" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-            </location>
-        </source>
-    </sources>
-    <instances>
-        <instance familyname="DesignspaceTest FamilyName" filename="out/DesignspaceTestFamilyName-Regular.ufo" name="DesignspaceTest FamilyName Regular" stylemapfamilyname="DesignspaceTest FamilyName" stylemapstylename="regular" stylename="Regular">
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="Custom Family" filename="out/CustomFamily-Regular.ufo" name="Custom Family Regular" stylemapfamilyname="Custom Family" stylemapstylename="regular" stylename="Regular">
-            <location>
-                <dimension name="Weight" xvalue="151" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-    </instances>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="400" maximum="600" default="400">
+      <map input="400" output="90"/>
+      <map input="600" output="151"/>
+    </axis>
+  </axes>
+  <sources>
+    <source filename="DesignspaceTestFamilyName-Regular.ufo" name="DesignspaceTest FamilyName Regular" familyname="DesignspaceTest FamilyName" stylename="Regular">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestFamilyName-Black.ufo" name="DesignspaceTest FamilyName Black" familyname="DesignspaceTest FamilyName" stylename="Black">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+      </location>
+    </source>
+  </sources>
+  <instances>
+    <instance name="DesignspaceTest FamilyName Regular" familyname="DesignspaceTest FamilyName" stylename="Regular" filename="out/DesignspaceTestFamilyName-Regular.ufo" stylemapfamilyname="DesignspaceTest FamilyName" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </instance>
+    <instance name="Custom Family Regular" familyname="Custom Family" stylename="Regular" filename="out/CustomFamily-Regular.ufo" stylemapfamilyname="Custom Family" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="151"/>
+      </location>
+    </instance>
+  </instances>
 </designspace>

--- a/tests/data/DesignspaceTestFileName.designspace
+++ b/tests/data/DesignspaceTestFileName.designspace
@@ -1,41 +1,37 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="400" maximum="600" minimum="400" name="Weight" tag="wght">
-            <map input="400" output="90" />
-            <map input="600" output="151" />
-        </axis>
-    </axes>
-    <sources>
-        <source familyname="DesignspaceTest FamilyName" filename="DesignspaceTestFamilyName-Regular.ufo" name="DesignspaceTest FamilyName Regular" stylename="Regular">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest FamilyName" filename="DesignspaceTestFamilyName-Black.ufo" name="DesignspaceTest FamilyName Black" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-            </location>
-        </source>
-    </sources>
-    <instances>
-        <instance familyname="DesignspaceTest FamilyName" filename="out/DesignspaceTestFamilyName-Regular.ufo" name="DesignspaceTest FamilyName Regular" stylemapfamilyname="DesignspaceTest FamilyName" stylemapstylename="regular" stylename="Regular">
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest FamilyName" filename="out/Custom FileName.ufo" name="DesignspaceTest FamilyName Regular" stylemapfamilyname="DesignspaceTest FamilyName" stylemapstylename="regular" stylename="Regular">
-            <location>
-                <dimension name="Weight" xvalue="151" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-    </instances>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="400" maximum="600" default="400">
+      <map input="400" output="90"/>
+      <map input="600" output="151"/>
+    </axis>
+  </axes>
+  <sources>
+    <source filename="DesignspaceTestFamilyName-Regular.ufo" name="DesignspaceTest FamilyName Regular" familyname="DesignspaceTest FamilyName" stylename="Regular">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestFamilyName-Black.ufo" name="DesignspaceTest FamilyName Black" familyname="DesignspaceTest FamilyName" stylename="Black">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+      </location>
+    </source>
+  </sources>
+  <instances>
+    <instance name="DesignspaceTest FamilyName Regular" familyname="DesignspaceTest FamilyName" stylename="Regular" filename="out/DesignspaceTestFamilyName-Regular.ufo" stylemapfamilyname="DesignspaceTest FamilyName" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest FamilyName Regular" familyname="DesignspaceTest FamilyName" stylename="Regular" filename="out/Custom FileName.ufo" stylemapfamilyname="DesignspaceTest FamilyName" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="151"/>
+      </location>
+    </instance>
+  </instances>
 </designspace>

--- a/tests/data/DesignspaceTestInactive.designspace
+++ b/tests/data/DesignspaceTestInactive.designspace
@@ -1,33 +1,31 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="600" maximum="600" minimum="600" name="Weight" tag="wght">
-            <map input="600" output="128" />
-        </axis>
-    </axes>
-    <sources>
-        <source familyname="DesignspaceTest Inactive" filename="DesignspaceTestInactive-Regular.ufo" name="DesignspaceTest Inactive Regular" stylename="Regular">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest Inactive" filename="DesignspaceTestInactive-Black.ufo" name="DesignspaceTest Inactive Black" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-            </location>
-        </source>
-    </sources>
-    <instances>
-        <instance familyname="DesignspaceTest Inactive" filename="out/DesignspaceTestInactive-Semibold.ufo" name="DesignspaceTest Inactive Semibold" stylemapfamilyname="DesignspaceTest Inactive Semibold" stylemapstylename="regular" stylename="Semibold">
-            <location>
-                <dimension name="Weight" xvalue="128" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-    </instances>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="600" maximum="600" default="600">
+      <map input="600" output="128"/>
+    </axis>
+  </axes>
+  <sources>
+    <source filename="DesignspaceTestInactive-Regular.ufo" name="DesignspaceTest Inactive Regular" familyname="DesignspaceTest Inactive" stylename="Regular">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestInactive-Black.ufo" name="DesignspaceTest Inactive Black" familyname="DesignspaceTest Inactive" stylename="Black">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+      </location>
+    </source>
+  </sources>
+  <instances>
+    <instance name="DesignspaceTest Inactive Semibold" familyname="DesignspaceTest Inactive" stylename="Semibold" filename="out/DesignspaceTestInactive-Semibold.ufo" stylemapfamilyname="DesignspaceTest Inactive Semibold" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="128"/>
+      </location>
+    </instance>
+  </instances>
 </designspace>

--- a/tests/data/DesignspaceTestInstanceOrder.designspace
+++ b/tests/data/DesignspaceTestInstanceOrder.designspace
@@ -1,49 +1,43 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="400" maximum="900" minimum="400" name="Weight" tag="wght">
-            <map input="400" output="90" />
-            <map input="700" output="151" />
-            <map input="900" output="190" />
-        </axis>
-    </axes>
-    <sources>
-        <source familyname="DesignspaceTest InstanceOrder" filename="DesignspaceTestInstanceOrder-Regular.ufo" name="DesignspaceTest InstanceOrder Regular" stylename="Regular">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest InstanceOrder" filename="DesignspaceTestInstanceOrder-Black.ufo" name="DesignspaceTest InstanceOrder Black" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-            </location>
-        </source>
-    </sources>
-    <instances>
-        <instance familyname="DesignspaceTest InstanceOrder" filename="out/DesignspaceTestInstanceOrder-Black.ufo" name="DesignspaceTest InstanceOrder Black" stylemapfamilyname="DesignspaceTest InstanceOrder Black" stylemapstylename="regular" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest InstanceOrder" filename="out/DesignspaceTestInstanceOrder-Regular.ufo" name="DesignspaceTest InstanceOrder Regular" stylemapfamilyname="DesignspaceTest InstanceOrder" stylemapstylename="regular" stylename="Regular">
-            <location>
-                <dimension name="Weight" xvalue="90" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest InstanceOrder" filename="out/DesignspaceTestInstanceOrder-Bold.ufo" name="DesignspaceTest InstanceOrder Bold" stylemapfamilyname="DesignspaceTest InstanceOrder" stylemapstylename="bold" stylename="Bold">
-            <location>
-                <dimension name="Weight" xvalue="151" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-    </instances>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="400" maximum="900" default="400">
+      <map input="400" output="90"/>
+      <map input="700" output="151"/>
+      <map input="900" output="190"/>
+    </axis>
+  </axes>
+  <sources>
+    <source filename="DesignspaceTestInstanceOrder-Regular.ufo" name="DesignspaceTest InstanceOrder Regular" familyname="DesignspaceTest InstanceOrder" stylename="Regular">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestInstanceOrder-Black.ufo" name="DesignspaceTest InstanceOrder Black" familyname="DesignspaceTest InstanceOrder" stylename="Black">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+      </location>
+    </source>
+  </sources>
+  <instances>
+    <instance name="DesignspaceTest InstanceOrder Black" familyname="DesignspaceTest InstanceOrder" stylename="Black" filename="out/DesignspaceTestInstanceOrder-Black.ufo" stylemapfamilyname="DesignspaceTest InstanceOrder Black" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest InstanceOrder Regular" familyname="DesignspaceTest InstanceOrder" stylename="Regular" filename="out/DesignspaceTestInstanceOrder-Regular.ufo" stylemapfamilyname="DesignspaceTest InstanceOrder" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest InstanceOrder Bold" familyname="DesignspaceTest InstanceOrder" stylename="Bold" filename="out/DesignspaceTestInstanceOrder-Bold.ufo" stylemapfamilyname="DesignspaceTest InstanceOrder" stylemapstylename="bold">
+      <location>
+        <dimension name="Weight" xvalue="151"/>
+      </location>
+    </instance>
+  </instances>
 </designspace>

--- a/tests/data/DesignspaceTestTwoAxes.designspace
+++ b/tests/data/DesignspaceTestTwoAxes.designspace
@@ -1,115 +1,101 @@
-<?xml version='1.0' encoding='utf-8'?>
-<designspace format="4.1">
-    <axes>
-        <axis default="400" maximum="900" minimum="100" name="Weight" tag="wght">
-            <map input="100" output="26" />
-            <map input="400" output="90" />
-            <map input="600" output="128" />
-            <map input="900" output="190" />
-        </axis>
-        <axis default="100" maximum="100" minimum="62.500000" name="Width" tag="wdth">
-            <map input="62.500000" output="70" />
-            <map input="100" output="100" />
-        </axis>
-    </axes>
-    <sources>
-        <source familyname="DesignspaceTest TwoAxes" filename="DesignspaceTestTwoAxes-Regular.ufo" name="DesignspaceTest TwoAxes Regular" stylename="Regular">
-            <lib copy="1" />
-            <groups copy="1" />
-            <features copy="1" />
-            <info copy="1" />
-            <location>
-                <dimension name="Weight" xvalue="90" />
-                <dimension name="Width" xvalue="100" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest TwoAxes" filename="DesignspaceTestTwoAxes-Black.ufo" name="DesignspaceTest TwoAxes Black" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-                <dimension name="Width" xvalue="100" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest TwoAxes" filename="DesignspaceTestTwoAxes-Thin.ufo" name="DesignspaceTest TwoAxes Thin" stylename="Thin">
-            <location>
-                <dimension name="Weight" xvalue="26" />
-                <dimension name="Width" xvalue="100" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest TwoAxes" filename="DesignspaceTestTwoAxes-ExtraCond.ufo" name="DesignspaceTest TwoAxes ExtraCond" stylename="ExtraCond">
-            <location>
-                <dimension name="Weight" xvalue="90" />
-                <dimension name="Width" xvalue="70" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest TwoAxes" filename="DesignspaceTestTwoAxes-ExtraCondBlack.ufo" name="DesignspaceTest TwoAxes ExtraCond Black" stylename="ExtraCond Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-                <dimension name="Width" xvalue="70" />
-            </location>
-        </source>
-        <source familyname="DesignspaceTest TwoAxes" filename="DesignspaceTestTwoAxes-ExtraCondThin.ufo" name="DesignspaceTest TwoAxes ExtraCond Thin" stylename="ExtraCond Thin">
-            <location>
-                <dimension name="Weight" xvalue="26" />
-                <dimension name="Width" xvalue="70" />
-            </location>
-        </source>
-    </sources>
-    <instances>
-        <instance familyname="DesignspaceTest TwoAxes" filename="out/DesignspaceTestTwoAxes-Thin.ufo" name="DesignspaceTest TwoAxes Thin" stylemapfamilyname="DesignspaceTest TwoAxes Thin" stylemapstylename="regular" stylename="Thin">
-            <location>
-                <dimension name="Weight" xvalue="26" />
-                <dimension name="Width" xvalue="100" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest TwoAxes" filename="out/DesignspaceTestTwoAxes-Regular.ufo" name="DesignspaceTest TwoAxes Regular" stylemapfamilyname="DesignspaceTest TwoAxes" stylemapstylename="regular" stylename="Regular">
-            <location>
-                <dimension name="Weight" xvalue="90" />
-                <dimension name="Width" xvalue="100" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest TwoAxes" filename="out/DesignspaceTestTwoAxes-Semibold.ufo" name="DesignspaceTest TwoAxes Semibold" stylemapfamilyname="DesignspaceTest TwoAxes Semibold" stylemapstylename="regular" stylename="Semibold">
-            <location>
-                <dimension name="Weight" xvalue="128" />
-                <dimension name="Width" xvalue="100" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest TwoAxes" filename="out/DesignspaceTestTwoAxes-Black.ufo" name="DesignspaceTest TwoAxes Black" stylemapfamilyname="DesignspaceTest TwoAxes Black" stylemapstylename="regular" stylename="Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-                <dimension name="Width" xvalue="100" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest TwoAxes" filename="out/DesignspaceTestTwoAxes-ExtraCondensedThin.ufo" name="DesignspaceTest TwoAxes ExtraCondensed Thin" stylemapfamilyname="DesignspaceTest TwoAxes ExtraCondensed Thin" stylemapstylename="regular" stylename="ExtraCondensed Thin">
-            <location>
-                <dimension name="Weight" xvalue="26" />
-                <dimension name="Width" xvalue="70" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest TwoAxes" filename="out/DesignspaceTestTwoAxes-ExtraCondensed.ufo" name="DesignspaceTest TwoAxes ExtraCondensed" stylemapfamilyname="DesignspaceTest TwoAxes ExtraCondensed" stylemapstylename="regular" stylename="ExtraCondensed">
-            <location>
-                <dimension name="Weight" xvalue="90" />
-                <dimension name="Width" xvalue="70" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-        <instance familyname="DesignspaceTest TwoAxes" filename="out/DesignspaceTestTwoAxes-ExtraCondensedBlack.ufo" name="DesignspaceTest TwoAxes ExtraCondensed Black" stylemapfamilyname="DesignspaceTest TwoAxes ExtraCondensed Black" stylemapstylename="regular" stylename="ExtraCondensed Black">
-            <location>
-                <dimension name="Weight" xvalue="190" />
-                <dimension name="Width" xvalue="70" />
-            </location>
-            <kerning />
-            <info />
-        </instance>
-    </instances>
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="100" maximum="900" default="400">
+      <map input="100" output="26"/>
+      <map input="400" output="90"/>
+      <map input="600" output="128"/>
+      <map input="900" output="190"/>
+    </axis>
+    <axis tag="wdth" name="Width" minimum="62.5" maximum="100" default="100">
+      <map input="62.5" output="70"/>
+      <map input="100" output="100"/>
+    </axis>
+  </axes>
+  <sources>
+    <source filename="DesignspaceTestTwoAxes-Regular.ufo" name="DesignspaceTest TwoAxes Regular" familyname="DesignspaceTest TwoAxes" stylename="Regular">
+      <lib copy="1"/>
+      <groups copy="1"/>
+      <features copy="1"/>
+      <info copy="1"/>
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+        <dimension name="Width" xvalue="100"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestTwoAxes-Black.ufo" name="DesignspaceTest TwoAxes Black" familyname="DesignspaceTest TwoAxes" stylename="Black">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+        <dimension name="Width" xvalue="100"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestTwoAxes-Thin.ufo" name="DesignspaceTest TwoAxes Thin" familyname="DesignspaceTest TwoAxes" stylename="Thin">
+      <location>
+        <dimension name="Weight" xvalue="26"/>
+        <dimension name="Width" xvalue="100"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestTwoAxes-ExtraCond.ufo" name="DesignspaceTest TwoAxes ExtraCond" familyname="DesignspaceTest TwoAxes" stylename="ExtraCond">
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+        <dimension name="Width" xvalue="70"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestTwoAxes-ExtraCondBlack.ufo" name="DesignspaceTest TwoAxes ExtraCond Black" familyname="DesignspaceTest TwoAxes" stylename="ExtraCond Black">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+        <dimension name="Width" xvalue="70"/>
+      </location>
+    </source>
+    <source filename="DesignspaceTestTwoAxes-ExtraCondThin.ufo" name="DesignspaceTest TwoAxes ExtraCond Thin" familyname="DesignspaceTest TwoAxes" stylename="ExtraCond Thin">
+      <location>
+        <dimension name="Weight" xvalue="26"/>
+        <dimension name="Width" xvalue="70"/>
+      </location>
+    </source>
+  </sources>
+  <instances>
+    <instance name="DesignspaceTest TwoAxes Thin" familyname="DesignspaceTest TwoAxes" stylename="Thin" filename="out/DesignspaceTestTwoAxes-Thin.ufo" stylemapfamilyname="DesignspaceTest TwoAxes Thin" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="26"/>
+        <dimension name="Width" xvalue="100"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest TwoAxes Regular" familyname="DesignspaceTest TwoAxes" stylename="Regular" filename="out/DesignspaceTestTwoAxes-Regular.ufo" stylemapfamilyname="DesignspaceTest TwoAxes" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+        <dimension name="Width" xvalue="100"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest TwoAxes Semibold" familyname="DesignspaceTest TwoAxes" stylename="Semibold" filename="out/DesignspaceTestTwoAxes-Semibold.ufo" stylemapfamilyname="DesignspaceTest TwoAxes Semibold" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="128"/>
+        <dimension name="Width" xvalue="100"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest TwoAxes Black" familyname="DesignspaceTest TwoAxes" stylename="Black" filename="out/DesignspaceTestTwoAxes-Black.ufo" stylemapfamilyname="DesignspaceTest TwoAxes Black" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+        <dimension name="Width" xvalue="100"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest TwoAxes ExtraCondensed Thin" familyname="DesignspaceTest TwoAxes" stylename="ExtraCondensed Thin" filename="out/DesignspaceTestTwoAxes-ExtraCondensedThin.ufo" stylemapfamilyname="DesignspaceTest TwoAxes ExtraCondensed Thin" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="26"/>
+        <dimension name="Width" xvalue="70"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest TwoAxes ExtraCondensed" familyname="DesignspaceTest TwoAxes" stylename="ExtraCondensed" filename="out/DesignspaceTestTwoAxes-ExtraCondensed.ufo" stylemapfamilyname="DesignspaceTest TwoAxes ExtraCondensed" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="90"/>
+        <dimension name="Width" xvalue="70"/>
+      </location>
+    </instance>
+    <instance name="DesignspaceTest TwoAxes ExtraCondensed Black" familyname="DesignspaceTest TwoAxes" stylename="ExtraCondensed Black" filename="out/DesignspaceTestTwoAxes-ExtraCondensedBlack.ufo" stylemapfamilyname="DesignspaceTest TwoAxes ExtraCondensed Black" stylemapstylename="regular">
+      <location>
+        <dimension name="Weight" xvalue="190"/>
+        <dimension name="Width" xvalue="70"/>
+      </location>
+    </instance>
+  </instances>
 </designspace>

